src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -379,56 +379,18 @@ public static long getJSONLong(JSONObject object, String key) {
 
 	/************* LDAP lookup ****************/
 	private static final String OU_GROUPS = "ou=user groups,ou=groups,dc=umich,dc=edu";
-	private static final String ALLOW_USER_URLOVERRIDE = "allow.testUser.urlOverride";
+	private static final String ALLOW_USER_URL = "use.testUser.url";
```

Comment:
I agree that the original name "allow.testUser.urlOverride" is more self-explanatory. I think we can repurpose it, and do not need to change the application.properties file.